reach/reach.py

```diff
@@ -628,6 +628,48 @@ def _most_similar_batch(
                 word_index = np.flip(np.argsort(sims_for_word))
                 yield [(self.indices[indices[idx]], sims_for_word[idx]) for idx in word_index]
 
+    def indices_threshold(
+        self,
+        vectors: npt.NDArray,
+        threshold: float = 0.5,
+        batch_size: int = 100,
+        show_progressbar: bool = False,
+    ) -> Iterator[np.ndarray]:
+        """
+        Retrieve the unsorted indices of the nearest neighbors for the given vectors above a threshold.
```

Comment:
Also leave a note referring to the original function somewhere. e.g., `if you also want to retrieve items and do not care about retrieval time as much, use ..`.